.changelog/23693.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+networking: Fixed a bug where if a client have two IP on the same network host, only one can provide the same static port
```

Comment:
It's a little difficult to understand the improved outcome here.  how about this?

```suggestion
networking: The same static port can now be used more than once on host networks with multiple IPs
```

Reply:
I had trouble to express it too, thank you for the wording